sr_demos/scripts/sr_demos/demo.py

```diff
@@ -45,9 +45,7 @@
 
 
 class TactileReading:
-    def __init__(self, hand_commander, demo_joint_states, prefix):
-        self.hand_commander = hand_commander
-        self.demo_joint_states = demo_joint_states
+    def __init__(self, prefix):
```

Comment:
Add pydoc please

Reply:
fixed in 92637be0ff846ca31b76aae33b809ec2e931900b